main.go

```diff
@@ -81,7 +81,7 @@ func main() {
 
 ### :ballot_box_with_check: Selection Criteria:
 
-- No reivews and pull requests was created within 24hrs
+- No reviews or pull requests were created in the last 24hrs
```

Comment:
The condition is really `&&`, if there's been reviews (other than approvals) it won't be listed

```suggestion
- No reviews and created in the last 24hrs
```

Reply:
No point in arguing with *the author*. :wink:

Reply:
English is not my first language I appreciate they help =D